.idea/fluentui-react-native.iml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We should probably delete all the .idea/ files. Nothing against jetbarains, but these are generated files we don't necessarily want as source and I think most use VS / VS Code for this repo anyway.

If you do want to keep using Jetbrains stufff, then I would add `.idea/*` to our gitignore file so git ignores adding them.

Reply:
Just deleted.